src/libraries/System.Private.Xml/src/Resources/Strings.resx

```diff
@@ -1722,7 +1722,7 @@
     <value>It is an error if 'length' is among the members of {facets} of {base type definition} and {value} is greater than the {value} of the parent 'length'.</value>
   </data>
   <data name="Sch_MinLengthGtBaseMinLength" xml:space="preserve">
-    <value>It is an error if 'minLength' is among the members of {facets} of {base type definition} and {value} is less than the {value} of the parent 'minLength'.</value>
+    <value>It is an error if the derived 'minLength' facet value is less than the parent 'minLength' facet value.</value>
```

Comment:
At this point it is anyone's guess what the original developer intended by these odd curlies. One possibility is that s/he planned to include the values instead of `{value}` ie 
`<value>It is an error if the derived 'minLength' facet value {0} is less than the parent 'minLength' facet value {1}.</value>`
and modifying the code to pass those in.

I think we would do this if this was an often seen string. However it has taken years for anyone to notice and report this bug, so it's apparently not a common string. I think updating them as you have done, without updating code to pass anything in, is fine.

Reply:
Agree with Dan, plus looks better for me without `facet`, e.g `It is an error if the derived 'minLength' value {0} is less than the parent 'minLength' value {1}`. From the code section you mentioned of the calling routine you cans pass those values directly, something like `throw new XmlSchemaException(SR.Sch_MinLengthGtBaseMinLength, _derivedRestriction.MinLength, _datatype.Restriction.MinLength);`